crates/rpc/rpc/src/eth/api/transactions.rs

```diff
@@ -1014,6 +1115,9 @@ pub(crate) fn build_transaction_receipt_with_block_receipts(
     meta: TransactionMeta,
     receipt: Receipt,
     all_receipts: &[Receipt],
+    #[cfg(feature = "optimism")] l1_block_info: &Option<L1BlockInfo>,
+    #[cfg(feature = "optimism")] l1_fee: Option<U256>,
+    #[cfg(feature = "optimism")] data_gas: Option<U256>,
```

Comment:
let's also group these into a helper type like OptimismBlockMeta or something

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/150